client/src/locales/en/translation.json

```diff
@@ -129,9 +129,12 @@
         "ExportButtonLabel": "Export",
         "reports":{
             "Title":"Reports",
-            "NameLabel":"Report Name",
-            "DateLabel":"Report Date",
-            "EmployeeLabel":"Concerned Employee(s)",
```

Comment:
Why was this removed in hoth translation files?

Reply:
I added them back.